assets/Discord-ServiceContract.md

```diff
@@ -0,0 +1,43 @@
+# Discord
+
+## Service Manager: _Chris Daniel_
+### Lifecycle: _Live_
+### Evolution: _Product_
+
+## Service Description
+_Discord is the easiest way to talk over voice, video, and text. Talk, chat, hang out, and stay close with other members of the Wardley Mapping Community._
+
+_Discord, operated by the foundation, will gather community members in one place and boost interaction and collaboration._
+
+_To create your account, follow [this link](https://discord.com/invite/upg2deYD3Z) and let the Discord app guide you._
+
+_If you need a new community channel, request one by opening issue in [this Github repository](https://github.com/Wardley-Mapping-Foundation/wmf-operating-model)._
+
+_When any assistance is needed, notify @admin or @moderator in a chat._
+
+
+## User Needs
+As a _newcomer_  
+I need _to ask a question about Wardley Mapping_  
+so that _I can move forward with my issue_
+
+As an _active community member_  
+I need _a place where I can meet mapping experts_  
+so that _I can discuss my vision for the mapping community_
+
+As a _potential workgroup manager_  
+I need _to have one discoverable place where I can meet with other working group members_  
+so that _the working group can easily assemble and communicate with each other_
+
+Metrics:
+ - _Discord metrics will be unlocked once we reach 500 members._
+
+
+## Service Measures
+ - _Discord metrics will be unlocked once we reach 500 members._
+
+## Service Metrics
```

Comment:
Looks like this is a duplicate in my template!  only need one of "Service Measures" or "Service Metrics".